vsphere/assets/configuration/spec.yaml

```
@@ -387,6 +387,13 @@ files:
         value:
           type: boolean
           example: false
+      - name: get_vsan
+        description: |
+          If true, the vSphere integration will collect additional metrics about vSAN cluster, host, and disk
+          metrics/events. These metrics/events are collected at a frequency of `refresh_infrastructure_cache_interval`.
```

```suggestion
          If true, the vSphere integration will collect additional metrics and events about vSAN clusters, hosts, and disks.
```
these are not collected at a frequency of refresh_infrastructure_cache_interval

---

changed.